todocore/src/main/java/com/knighttodo/todocore/rest/DayTodoResource.java

```diff
@@ -175,4 +177,32 @@ public DayTodoReadyResponseDto updateIsReady(@PathVariable UUID dayId, @PathVari
             throw new DayTodoReadyCanNotBeUpdatedException("Day todo ready can't be updated.", ex);
         }
     }
+
+    @PatchMapping
+    @ResponseStatus(HttpStatus.CREATED)
```

Comment:
we don't create we update it here. OK status will be good

Reply:
I would like understand with statuses.